common/djangoapps/util/sandboxing.py

```diff
@@ -32,8 +30,9 @@ def can_execute_unsafe_code(course_id):
 
 
 def get_python_lib_zip(contentstore, course_id):
```

Comment:
It's admirable to make this "course code library" name configurable - but this method is specifically named. Should this method name be changed as well?

Reply:
Good catch. Will do 👍

Reply:
Ended up changing the "course code library" references to "python lib" (et al). "Python lib" was already a convention in other parts of the codebase. It was easier to just center around that